src/main/java/org/broadinstitute/hellbender/tools/exome/sexgenotyper/TargetCoverageSexGenotyper.java

```diff
@@ -0,0 +1,172 @@
+package org.broadinstitute.hellbender.tools.exome.sexgenotyper;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.broadinstitute.hellbender.cmdline.Argument;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.CopyNumberProgramGroup;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.exome.ReadCountCollection;
+import org.broadinstitute.hellbender.tools.exome.ReadCountCollectionUtils;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.param.ParamUtils;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * A command line tool for inferring sex genotypes from a tab-separated raw target coverage file.
+ *
+ * <p>
+ *     In addition to the raw target coverage file, the user must provide a tab-separated contig
+ *     ploidy annotation file. For homo sapiens, the annotation file may be as follows:
+ *
+ *     <pre>
+ *         CONTIG_NAME    CONTIG_CLASS    SEX_XX    SEX_XY
+ *         1              AUTOSOMAL       2          2
+ *         2              AUTOSOMAL       2          2
+ *                                   ...
+ *                                   ...
+ *                                   ...
+ *         X              ALLOSOMAL       2          0
+ *         Y              ALLOSOMAL       1          1
+ *     </pre>
+ *
+ *     CONTIG_NAME must be same as contig names that appear in the read counts table. CONTIG_CLASS is
```

Comment:
appears refers to contig names. Updated the doc to `CONTIG_NAME column values must be same as contig names that appear in the read counts table.`